app/proc/store.go

```diff
@@ -35,7 +35,8 @@ func NewBoltDB(dbFile string) (*BoltDB, error) {
 }
 
 // Save to bolt, skip if found
-func (b BoltDB) Save(fmFeed string, item feed.Item) error {
+func (b BoltDB) Save(fmFeed string, item feed.Item) (bool, error) {
```

Comment:
having hard time to figure this one out. Why created bool is even needed?  Isn't error enough for this? I mean in what situation you may get err=nil and created=false?

Reply:
> Why created bool is even needed?

if `key` exists `bucket` i'm not send notification telegram, only if is new key

Reply:
`created` use here
[4b27d353bc4e545aa15acb5ea491dea4e8a92800..b5e57313cd720101c0aaa987e84d945cc80ac9fc](https://github.com/umputun/feed-master/pull/14/files/4b27d353bc4e545aa15acb5ea491dea4e8a92800..b5e57313cd720101c0aaa987e84d945cc80ac9fc#diff-ddba86681f43d87fb304f116883ae19fR96-R104)

Reply:
> if key exists bucket i'm not send notification telegram, only if is new key

ok, I get it. But the logic is a little bit awkward. I think changing it to return some KeyAlreadyExistsError makes more sense.

Another thing - your changes lack any tests. Yeah, I know this project is not covered well and other parts missing tests, but I'm planning to deal with this someday and it would be nice to have parts you touched/added covered from the day one.

Reply:
write tests in current PR?